src/components/messageCanvas/messageCanvas.stories.tsx

```diff
@@ -81,7 +64,7 @@ const profileString = `(message: ThreadableMessage) => {
   }`
 
 const getProfileComponent = (message: ThreadableMessage) => {
-  if (message.sender.includes('agent')) {
+  if (message.sender.name.includes('agent')) {
```

Comment:
Name is optional, so `?` is needed. Could you also update the `profileString` above it to use `message.sender.name?.includes('agent')`
```suggestion
  if (message.sender.name?.includes('agent')) {
```

Reply:
nice catch